### ISSUE TITLE
Webapps use the same entry/icon as Firefox in the dock

### ISSUE THREAD

**Issue by spazziale** [cosmic-utils/web-apps#48]
de: Gnome 46 on wayland
os: fedora

Firefox and webapps should be separate, each with its own icons. 
![image](https://github.com/elevenhsoft/WebApps/assets/46284310/8b32d428-4e6b-42bb-9598-61e3677ff14d)

**Comment by elevenhsoft** [comment]
Yeah it works in other desktops than GNOME so I'm not sure what's the case. We missing some entry in desktop file specific for GNOME or GNOME is GNOME and doing things his own way. If someone have idea, please let me know.

**Comment by elevenhsoft** [comment]
That's interesting. It won't hurt to add this also.

**Comment by hepp3n** [comment]
We are changing wmclass and also window name for each created app. Also, we are setting StartupWMClass to match this. GNOME can't handle this. On other desktops it works. I can't or don't know how to fix it. Let's close this.
